### PR TITLE
testing mesh with istio1.2

### PR DESCRIPTION
**Pull request by chaodaiG**
@Fredy-Z 

/hold
Need to push image for Flaky test reporter and update Prow after this PR merged

### CODE REVIEW

File: tools/flaky-test-reporter/config.go [hunk 1]
```diff
@@ -39,10 +39,12 @@ var (
 	jobConfigs = map[string][]JobConfig{
 		// CI flows for serving repo
 		"serving": {{Name: "ci-knative-serving-continuous", Type: prow.PostsubmitJob, SkipGithubIssue: false},
-			{Name: "ci-knative-serving-istio-1.0.7-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.0.7-no-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.1.7-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
-			{Name: "ci-knative-serving-istio-1.1.7-no-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
+			{Name: "ci-knative-serving-istio-1.0-mesh", Type: prow.PostsubmitJob, SkipGithubIssue: true},
```
Comment (adrcunha) [inline tools/flaky-test-reporter/config.go]: Wow. This definitely should be a dynamic config parser... Someday.

Comment (chaodaiG) [reply tools/flaky-test-reporter/config.go]: will be

**Pull request merged**